planwise.iml

```diff
@@ -206,7 +208,7 @@
     <orderEntry type="library" name="Leiningen: ragtime/core:0.7.2" level="project" />
     <orderEntry type="library" name="Leiningen: ragtime/jdbc:0.7.2" level="project" />
     <orderEntry type="library" name="Leiningen: ragtime:0.7.2" level="project" />
-    <orderEntry type="library" name="Leiningen: re-com:2.1.0" level="project" />
+    <orderEntry type="library" name="Leiningen: re-com:0.9.0" level="project" />
```

Comment:
@ftarulla watchout this rollback

Reply:
@bcardiff  Yes! I noticed the same thing. I searched for re-frame / re-com and I found that the master version is 0.9.0 https://github.com/Day8/re-com

Reply:
outdated README! 😞 ... Fixing .iml re-com entry to match version 2.1.0